lib/sync.js

```diff
@@ -38,6 +38,14 @@ var maybeUnwrapSymlink = function maybeUnwrapSymlink(x, opts) {
     return x;
 };
 
+var getPackageCandidates = function getPackageCandidates(x, start, opts) {
+    var dirs = nodeModulesPaths(start, opts, x);
```

Comment:
Since this function is the same in `sync.js`, maybe it could be extracted to a separate file?

Reply:
I thought about that, but then it's a separate file that has to be supported for the rest of v1.